File: Course_01/C++ Language/EstruturaRepetitiva/dentro_fora.cpp
#include <iostream>

using namespace std;

int main()
{
    int n, conteDentro(0), conteFora(0), i, x;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> x;

    for (i = 0; i < x; i++) {
        cout << "Digite um numero: ";
        cin >> n;
        if (n >= 10 &&  n <= 20) {
            conteDentro++;
        }
        else {
            conteFora++;
        }
    }
    cout << conteDentro << " DENTRO" << endl;
    cout << conteFora << " FORA" << endl;
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/dentro_fora.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class dentro_fora {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int i, x, n, conteDentro;
		
		System.out.print("Quantos numeros voce vai digitar? ");
		x = sc.nextInt();
		
		conteDentro = 0;
		
		for (i = 0; i < x; i++) {
			System.out.print("Digite um numero: ");
			n = sc.nextInt();
			if (n >= 10 && n <= 20) {
				conteDentro++;
			}
		}
		System.out.println(conteDentro + "DENTRO");
		System.out.println(x - conteDentro + "FORA");
		sc.close();
	}

}
